GoofyLightsEditor/FrameList.cpp

```diff
@@ -116,8 +117,7 @@ void FrameList::DeleteNode_Middle(int pos){
         // Delete the head node.
         head = current -> next;
         head -> prev = NULL;
-        // Delete Attached RGB structure here 
-        delete(current);
```

Comment:
So i think:
delete_RGB(head -> FrameData.data, this -> row, this -> col);
delete (current);

would fix this issue, this my also be what is causing the seg fault, which would only happen when calling this function.